src/scss/components/_media-element.scss

```diff
@@ -126,6 +126,11 @@ $mediaElement-spacing           : spacing(x2);
             margin-top: spacing(x0.5);
         }
 
+        .c-mediaElement--fullstack--negativeTop & {
+            flex: auto;
+            margin-top: -#{spacing(x2)} !important;
```

Comment:
Do you need `!important` here? This is declared after the previous rule, where margin was being applied, so the cascade should mean that this syle has precedence?